src/server/src/Server.cpp

```diff
@@ -201,6 +203,40 @@ std::string Server::track_position(const double x, const double y, const double
 }
 
 
+std::string Server::set_position_track_param(const double amax, const double vmax, const double jerk)
+{
+    State state = get_state();
+
+    if (state != State::Idle)
```

Comment:
What is the rationale of this and following lines?

If the state is different from `Idle` it is set to Idle. However, later you change it to `SetPosTrackParam`.

Reply:
see following answer